modules/ting_reference/ting_reference.module

```diff
@@ -206,7 +206,7 @@ function _ting_reference_field_update_cron($entity_type, $entity, &$items) {
 
   // Clear cache if needed.
   if ($clear_cache) {
-    field_cache_clear('field:' . $entity_type . ':' . $entity_id);
```

Comment:
Why has this targeted caching been changed?

Reply:
That a "issues" found in getting the ting_reference carousel's working on event and news nodes'. The clear field cache function don't take any parameters.

See https://api.drupal.org/api/drupal/modules%21field%21field.module/function/field_cache_clear/7.x

Reply:
:+1: